src/main/webapp/app/exam/participate/exercises/programming/code-editor/exam-code-editor-student-container.component.ts

```diff
@@ -63,21 +84,24 @@ export class ExamCodeEditorStudentContainerComponent extends CodeEditorContainer
     }
 
     reload(): void {
-        this.ngOnInit();
-        if (this.instructions) {
-            this.instructions.refreshInstructions();
-        }
+        // this.ngOnInit();
+        // if (this.instructions) {
+        //     this.instructions.refreshInstructions();
+        // }
```

Comment:
I would suggest to remove it if it is not needed anymore instead of commenting it out.

Reply:
This depends on the outcome of the reviews :D